src/index.ts

```diff
@@ -120,11 +120,20 @@ export const index = async (input: string = process.argv[2], options: IndexOptio
   ];
 
   const shouldRecheck = (status: Status, lastCheckedAt: string) => {
-    const shouldIndexIt = indexableStatuses.includes(status);
+    if (status !== Status.SubmittedAndIndexed) {
```

Comment:
this seems wrong, there are some statuses that should not force a recheck

or am I missing something?

Reply:
Ah you're right, I overfit to my own personal issue.

My cache was saving "status": "Not found (404)", which is not currently part of the Status enum, so those were being skipped.

I should rather just add a new Status.NotFound to indexableStatus.

However, such things can be fixed quite quickly but because of the long cacheTimeout they'd be ignored for 2 weeks. Perhaps we can have a shorter timeout for NotFound or PageWithRedirect. 

What do you think?